test/ttmlir/Silicon/multiple_add_with_loc.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline %s | ttmlir-translate --ttnn-to-flatbuffer > %s.ttnn
```

Comment:
I think that `%t.ttnn` might be what we want right?  Just as a rule of thumb we don't want build or test artifacts to show up in the source tree.